gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -54,8 +55,13 @@ public void apply(Project project) {
 
     private void applyToJavaProject(Project project) {
         if (isDisabled(project) && isAnyKindOfPatching(project)) {
-            throw new IllegalStateException(
-                    "-PerrorProneDisable cannot be used at the same time as -PerrorProneApply or -PerrorProneSuppress");
+            throw new IllegalStateException("-PerrorProneDisable cannot be used at the same time as "
+                    + "-PerrorProneApply, -PerrorProneSuppress or -PerrorProneRemoveRollout");
+        }
+
+        if (isRemovingSuppressions(project) && (isSuppressing(project) || isApplyingSuggestedPatches(project))) {
+            throw new IllegalStateException("-PerrorProneRemoveRollout cannot be used at the same time as "
+                    + "-PerrorProneApply or -PerrorProneSuppress");
```

Comment:
I could potentially try to make this possible, but this would lead to more complexity in the error prone options setup and it doesn't seem like something that is needed for now, hence why I didn't

Reply:
Yeah, makes sense to just keep it simple and make it error out. Unlikely to both be applying/suppressing and removing _a specific_ errorprone suppression at the same time.